internal/storage/image.go

```diff
@@ -49,6 +49,7 @@ type ImageResult struct {
 	PreviousName string
 	Labels       map[string]string
 	OCIConfig    *specs.Image
+	Intermediate bool
```

Comment:
I don't see this being used anywhere, looks like it is not needed.

Reply:
it's being used here, to write the info 
https://github.com/cri-o/cri-o/pull/3892/files#diff-401dc45799b30a238ffacfa0faed4248R100